{{cookiecutter.directory_name}}/docs/team/council.yml

```diff
@@ -0,0 +1,84 @@
+# Active Council members
+# Use ALPHABETICAL (BY LAST NAME) ORDER please :-)
+
+# - name: Joe/Jane
+#   handle: "@joe-jane"
+#   affiliation: Jupyter
+#   team: active
+#   last-check-in: 2022-1
+
+
+- name: Damián Avila
+  handle: "@damianavila"
+  affiliation: UMSI and 2i2c
+  team: active
+  last-check-in: 2022-03
+
+- name: Mehmet Bektas
+  handle: "@mbektas"
+  affiliation: Netflix
+  team: active
+  last-check-in: 2022-03
+
+- name: Martha Cryan
+  handle: "@marthacryan"
+  affiliation: IBM CODAIT
+  team: active
+  last-check-in: 2022-03
+
+- name: Afshin Darian
+  handle: "@afshin"
+  affiliation: Two Sigma
+  team: active
+  last-check-in: 2022-03
+
+- name: Brian Granger
+  handle: "@ellisonbg"
+  affiliation: Amazon Web Services
+  team: active
+  last-check-in: 2022-03
+
+- name: Jason Grout
+  handle: "@jasongrout"
+  affiliation: Databricks
+  team: active
+  last-check-in: 2022-03
+
+- name: Paul Ivanov
+  handle: "@ivanov"
+  affiliation: Noteable
+  team: active
+  last-check-in: 2022-03
+
+- name: Michał Krassowski
+  handle: "@krassowski"
+  affiliation: University of Oxford
+  team: active
+  last-check-in: 2022-03
+
+- name: Fernando Perez
+  handle: "@fperez"
+  affiliation: University of California, Berkeley
+  team: active
+  last-check-in: 2022-03
+
+- name: Isabela Presedo-Floyd
+  handle: "@isabela-pf"
+  affiliation: Quansight Labs
+  team: active
+  last-check-in: 2022-03
+
+- name: Steve Silvester
+  handle: "@blink1073"
+  affiliation: MongoDB
+  team: active
+  last-check-in: 2022-03
+
+- name: Jeremy Tuloup
+  handle: "@jtpio"
+  affiliation: Quantstack
+  team: active
+  last-check-in: 2022-03
```

Comment:
```suggestion
```
Deleting the JupyterLab council members... I think the commen provides enough info for adding people.